webapp/src/websocket/index.js

```diff
@@ -66,21 +63,23 @@ export function handleReconnect(store, reminder = false) {
     return async () => {
         const {data} = await getConnected(reminder)(store.dispatch, store.getState);
         if (data && data.connected) {
-            getReviews()(store.dispatch, store.getState);
-            getUnreads()(store.dispatch, store.getState);
-            getYourPrs()(store.dispatch, store.getState);
-            getYourAssignments()(store.dispatch, store.getState);
+            const rand = Math.floor(Math.random() * (JitterForReconnectAPICall.MAX_TIME - JitterForReconnectAPICall.MIN_TIME + 1)) + JitterForReconnectAPICall.MIN_TIME; //eslint-disable-line no-mixed-operators
+            setTimeout(() => {
+                getSidebarContent()(store.dispatch, store.getState);
+            }, rand);
         }
```

Comment:
store the ID of this timeout and clear it when the component unmounts and when `handleReconnect` is called again
It could be possible that a user/program calls `handleReconnect` multiple times so in that case if there is already any timeout running in the BG here we can clear that before placing a new one